### ISSUE TITLE
need help/instructions

### ISSUE THREAD

**Issue by sejo153** [davidmann4/ogar-feeder-bot#374]
I've tried a lot of cookies and diferent ways  trying to find way to fix "failed to get token" and i didnt find.
If anyone can help i would love him.

**Comment by hassanila** [comment]
**@sejo153 Do you use your own cookies?  If yes and you get token failed I know the solution:**

---

If the above is true then the problem is that you need to **NOT** logout from your FB when you get the cookies otherwise they will be invalidated (that's why you get **token failed**).
What you need todo is to get the cookies and instead of logging out you need to delete the cookies and refresh, in that way you can logout without invalidating the cookies.
**For deleting the cookies you need [**This extension (Chrome)**](https://chrome.google.com/webstore/detail/editthiscookie/fngmhnnpilhplaeedifhccceomclgfbg?hl=en)**

**Comment by sejo153** [comment]
Easy job. Thanks you so much!!

**Comment by ghost** [comment]
@hassanila97 All cookies or the c_user, datr and xs?

**Comment by sejo153** [comment]
Well, i've done job well and dont got "FAILED TO GET TOKEN" its okay its says "Total Tokens: number"
But big bots dont come

**Comment by hassanila** [comment]
@sejo153 Yes, it means the tokens are right and everything good but you don't see the bots cuz it connects it to one connection only and if that fails no mass bot. The fix is pretty easy... but needs some coding.

@BrunoOlyver For logging out? You need to delete all.   For feeder bots you need only:  c_user, datr and xs

---

I edited some code, now the bots work almost perfect...  I took over all server, noone could join  :=)  ( I have 0 in the counter cuz its screwed up)
![untitled](https://cloud.githubusercontent.com/assets/15068696/14235422/0af8fe32-f9fe-11e5-81e2-a461766f410e.png)

![2016-04-05 18_43_16-greenshot](https://cloud.githubusercontent.com/assets/15068696/14290385/2de9b848-fb5f-11e5-8e5e-d87ca9fd7aaa.png)

![2016-04-05 18_27_04-greenshot](https://cloud.githubusercontent.com/assets/15068696/14290391/32cb241e-fb5f-11e5-90f2-97adb221e4e4.png)

**Comment by sejo153** [comment]
You can help me out ?

**Comment by hassanila** [comment]
@sejo153  It's late now, gonna sleep.... I will update the code on this repo (PR) not now.

---

**UPDATE:**  I dont think I will.... Every1 would be destroying servers as it is optimized for maximum bots.. .. sry :(

**Comment by sejo153** [comment]
Okay. Thanks

**Comment by sejo153** [comment]
@hassanila97 Nice :D

**Comment by ghost** [comment]
OMG Thanks, more i create my facebook accounts with Microsoft , have others methods for no lose time?

**Comment by sejo153** [comment]
@hassanila97 could you upload code now ?

**Comment by hassanila** [comment]
I dont think I will....  Every1 would be destroying servers as it is optimized for maximum bots.. .. sry :(

**Comment by sejo153** [comment]
I thought this place is helping people...

**Comment by sejo153** [comment]
skype me if you change your mind sejo.dedic4

**Comment by dimamelnic** [comment]
@hassanila97 Hi. Can I get your bots?
